src/components/standalone/backup_and_restore/BackupContent.vue

```diff
@@ -0,0 +1,256 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { ubusCall } from '@/lib/standalone/ubus'
+import {
+  getAxiosErrorMessage,
+  NeButton,
+  NeEmptyState,
+  NeInlineNotification,
+  NeSkeleton
+} from '@nethserver/vue-tailwind-lib'
+import NeTable from '@/components/standalone/NeTable.vue'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import ModalDownloadBackup from '@/components/standalone/backup_and_restore/ModalDownloadBackup.vue'
+import ModalRunBackup from '@/components/standalone/backup_and_restore/ModalRunBackup.vue'
+import DrawerSetPassphrase from '@/components/standalone/backup_and_restore/DrawerSetPassphrase.vue'
+
+const { t } = useI18n()
+
+interface Backup {
+  file: string
+  name: string
+}
+
+const loading = ref(true)
+const loadingPassphrase = ref(true)
+const isEnterprise = ref(false)
+const isSetPassphrase = ref(false)
+const showDownloadModal = ref(false)
+const showPassphraseDrawer = ref(false)
+const showRunBackupModal = ref(false)
+const successNotificationRunBackup = ref(false)
+const unitName = ref()
+const seletedBackup = ref('')
+const listBackups = ref<Backup[]>([])
+
+let error = ref(false)
+let errorPage = ref({
+  notificationTitle: '',
+  notificationDescription: ''
+})
+
+onMounted(() => {
+  Promise.all([getSubscription(), getHostname(), getIsPassphrase()])
+})
```

Comment:
This won't work, promises are failing silently and no usage is taken from this, you can use this [like this example](https://github.com/NethServer/nethsecurity-ui/blob/a6a4ef4d90b531534cee1317d29241fa4210e850/src/composables/useMwan.ts#L46C27-L46C27), you don't need any returns in your case